appintro/src/main/res/layout/appintro_fragment_intro.xml

```diff
@@ -30,22 +30,27 @@
         android:layout_height="0dp"
         app:layout_constraintVertical_weight="5"
         android:contentDescription="@string/app_intro_image_content_description"
-        app:layout_constraintBottom_toTopOf="@+id/description"
+        app:layout_constraintBottom_toTopOf="@+id/scrollable_textview"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/title" />
 
-    <TextView
-        android:id="@+id/description"
-        style="@style/AppIntroDefaultText"
+    <ScrollView

```

Comment:
The scrollview is not needed. It should be enought with: 
```
android:scrollbars = "vertical"
```

Plus 

```java
textView.setMovementMethod(new ScrollingMovementMethod());
```

Reply:
OK, I will work on this right now. :) 
Thank you so much!

Reply:
Hi! Those two methods seem to have the same effect as far as I am concerned. And I also guess it would be easier to maintain the project if we let the XML do all the rending work. Editing Java files would also work, but maybe it is harder for other developers to associate it with the XML file. This is my opinion. I would appreciate it if you could give me some advice on this. Thank you so much! :)

Reply:
If it is OK with you I would like to do the adaptation work of the land mode. :)

Reply:
Let's go with the XML only approach 👍

Reply:
Please note this: 

> The scrollview is not needed.

You can remove the ScrollView entirely 👍

Reply:
Hi! Sorry for the wait. I guess I have solved the problem when the landscape mode is used. Here are some screenshots after modifications when using the largest display size:

<img src="https://user-images.githubusercontent.com/25502419/131349299-baa4fe16-94ae-4a55-9ac2-1312833ada2d.png" alt="copy" width="500"/>
<img src="https://user-images.githubusercontent.com/25502419/131349309-2849b21b-db58-421e-b737-6b088130d656.png" alt="copy" width="500"/> 
<img src="https://user-images.githubusercontent.com/25502419/131349318-93e41e15-b099-43ab-a1d4-8a0be7ce8630.png" alt="copy" width="500"/>

and the smallest display size:

<img src="https://user-images.githubusercontent.com/25502419/131349324-9f9492a2-9568-436e-b800-2ac499ce8847.png" alt="copy" width="500"/>
<img src="https://user-images.githubusercontent.com/25502419/131349329-18b8df72-6981-40cf-8b07-02c3602ba14a.png" alt="copy" width="500"/>

I am looking forward to your review. Thanks again for your kindness and your precious time!! :)

Reply:
`You can remove the ScrollView entirely`

If I remove the ScrollView, should I use the java code you recommended earlier?

Reply:
Please do your own researches/trial and let me know what's the outcome.
The solution with the least amount of changes is generally preferred.

Like if we can solve it by just adding an XML attribute, that would be the preferred approach.

Reply:
OK, I could comprehend the meaning you are trying to deliver to me now. Therefore I would like to make a comparison of those two methods and I would like to inform you of this once it is ready. And I will also minimize the changes I could apply to those files. :)
Thanks again!